axiom/nr_txn.c

```diff
@@ -3300,12 +3302,24 @@ static void log_event_set_linking_metadata(nr_log_event_t* e,
                                            nrapp_t* app) {
   char* trace_id = NULL;
   char* span_id = NULL;
+  nr_segment_t* segment = NULL;
 
   if (NULL == e) {
     return;
   }
 
+  // default priority to lowest value
+  nr_log_event_set_priority(e, 0);
+
   if (nrlikely(txn)) {
+    segment = nr_txn_get_current_segment(txn, NULL);
+    if (NULL != segment) {
+      // bump segment priority to increase chance it is saved
+      // if sampling occurs
+      nr_segment_set_priority_flag(segment, NR_SEGMENT_PRIORITY_LOG);
+      nr_log_event_set_priority(e, nr_segment_get_priority_flag(segment));
+    }
```

Comment:
```suggestion
  /* default priority to lowest value */
  nr_log_event_set_priority(e, 0);

  if (nrlikely(txn)) {
    segment = nr_txn_get_current_segment(txn, NULL);
    if (NULL != segment) {
      /*
      * bump segment priority to increase chance it is saved
      * if sampling occurs
      */
      nr_segment_set_priority_flag(segment, NR_SEGMENT_PRIORITY_LOG);
      nr_log_event_set_priority(e, nr_segment_get_priority_flag(segment));
    }
```
Another nit, but when we scrubbed for open source, we tried to move all comments to the `/* */` format.

Reply:
Thanks for the heads up on this.